requirements/base.in

```diff
@@ -1,6 +1,8 @@
 # Core requirements for using this package template
 -c constraints.txt
 
+drf-jwt<1.15.0  # max contraint added here to work with setup.py's `install_requires`.
```

Comment:
I agree with @robrap.  Since this library is intended for rest API clients only, adding a dependency on DRF seems out of place - and could lead to unnecessary maintenance woes.